src/main/java/com/google/api/codegen/ArtifactType.java

```diff
@@ -28,9 +28,13 @@
 
   // This will be split into GAPIC and GAPIC_PACKAGE
   LEGACY_GAPIC_AND_PACKAGE,
-  // This will be split into DISCOGAPIC and DISCOGAPIC_PACKAGE
+  // This will be split into DISCOGAPIC_CODE and DISCOGAPIC_PACKAGE
```

Comment:
The comment for `LEGACY_GAPIC_AND_PACKAGE` should be corrected too

Reply:
Updated, also changed "will be - is" (but not for discogapic, because there is no discogapic_package)